src/web/core/src/main/java/org/geoserver/web/StringAndInternationalStringPanel.java

```diff
@@ -55,4 +56,38 @@ protected TextField<String> getTextComponent(String id, IModel<String> model) {
 
         add(internationalStringField);
     }
+
+    public StringAndInternationalStringPanel(
+            String id,
+            IModel<?> model,
+            String labelKey,
+            String stringProperty,
+            String internationalProperty,
+            WebMarkupContainer labelProvider,
+            IValidator<String> validator) {
+        super(id, model);
+        WebMarkupContainer container = new WebMarkupContainer("labelContainer");
```

Comment:
The addition of validators is done in the right places, but duplicating the entire existing constructor to add them is overkill. I suggest using a single constructor with actual code, a couple of ifs, and the previous constructor would call the new one passing null as the validator.

Reply:
I removed the extra constructor as suggested